### PR TITLE
Sonarcloud Analysis once a week.

### PR DESCRIPTION
**Pull request by jflo**
Runs sonarqube analysis as a github action once a week, on Tuesday mornings at midnight, UTC. Memory requirements have increased to 4G heap.

## Documentation

- [X] I thought about documentation and added the `doc-change-required` label to this PR if
    [updates are required](https://wiki.hyperledger.org/display/BESU/Documentation).

## Acceptance Tests (Non Mainnet)

- [X] I have considered running `./gradlew acceptanceTestNonMainnet` locally if my PR affects non-mainnet modules.

## Changelog

- [X] I thought about the changelog and included a [changelog update if required](https://wiki.hyperledger.org/display/BESU/Changelog).

### CODE REVIEW

File: gradlew [hunk 1]
```diff
@@ -86,7 +86,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=${0##*/}
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS='"-Xmx64m" "-Xms64m"'
+DEFAULT_JVM_OPTS='"-Xmx1024m" "-Xms256m"'
```
Comment (siladu) [inline gradlew:89]: Isn't gradle.properties taking precedence and so this is unnecessary?

**Review by siladu — APPROVED** [review]
LGTM

**Review by macfarla — APPROVED** [review]

**Review by jframe — APPROVED** [review]

File: gradlew (hunk 1 above)
Comment (jflo) [reply gradlew:89]: Sadly, it is necessary due to out of memory conditions we encountered while compiling.

Comment (siladu) [reply gradlew:89]: I wonder if that will get blatted away the next time we update gradle 🤔

Comment (siladu) [reply gradlew:89]: ...looks like you might be about to find out :)
https://github.com/hyperledger/besu/pull/5144
cc @usmansaleem

**Pull request merged**